test/integration/targets/vyos_lag_interfaces/tests/cli/merged.yaml

```diff
@@ -0,0 +1,54 @@
+---
+- debug:
+      msg: "START vyos_lag_interfaces merged integration tests on connection={{ ansible_connection }}"
+
+- include_tasks: _remove_config.yaml
+
+- name: Merge the provided configuration with the exisiting running configuration
+  vyos_lag_interfaces: &merged
+      config:
+          - name: bond0
+            hash-policy: "layer2"
+            mode: "active-backup"
+            members:
+                - member: eth1
+            primary: eth1
+
+          - name: bond1
+            hash-policy: "layer2+3"
+            mode: "active-backup"
+            members:
+                - member: eth2
+            primary: eth2
+      state: merged
+  register: result
+
+- name: Assert that before dicts were correctly generated
```

Comment:
This appears to be failing, but I don't really understand the logic here

Reply:
We are comparing that config before  actual merged state  operation and config shown in output result ( 'before[]') after executing merge operation should be same.

Reply:
I suspect the before config is in the wrong state before we are running our test. It is hard to see what the issue is, but we should be able to debug this.

Reply:
Yeah seems like some sort of race condition , or delay . I am gonna try to explicitly set the state giving enough time for system to get ready for next state operation  and  yeah we  should be able to debug this.